Documentation/Storage-Configuration/Block-Storage-RBD/rbd-mirroring.md

```diff
@@ -165,13 +165,13 @@ APIs for storage disaster recovery. The extended APIs are provided via Custom
 Resource Definition(CRD). Create the VolumeReplication CRDs on all the peer clusters.
 
 ```console
-kubectl create -f https://raw.githubusercontent.com/csi-addons/volume-replication-operator/v0.3.0/config/crd/bases/replication.storage.openshift.io_volumereplications.yaml
-kubectl create -f https://raw.githubusercontent.com/csi-addons/volume-replication-operator/v0.3.0/config/crd/bases/replication.storage.openshift.io_volumereplicationclasses.yaml
+kubectl create -f https://raw.githubusercontent.com/csi-addons/kubernetes-csi-addons/main/config/crd/bases/replication.storage.openshift.io_volumereplications.yaml
+kubectl create -f https://raw.githubusercontent.com/csi-addons/kubernetes-csi-addons/main/config/crd/bases/replication.storage.openshift.io_volumereplicationclasses.yaml
```

Comment:
We will have a versioned link soon, right? Referencing main would be less stable.

Reply:
yes, once we have csi-addons release I will update them too.